docs/upgrade.html

```diff
@@ -70,6 +133,8 @@
     <li>The Zookeeper dependency was removed from the Streams API. The Streams API now uses the Kafka protocol to manage internal topics instead of
         modifying Zookeeper directly. This eliminates the need for privileges to access Zookeeper directly and "StreamsConfig.ZOOKEEPER_CONFIG"
         should not be set in the Streams app any more. If the Kafka cluster is secured, Streams apps must have the required security privileges to create new topics.</li>
+    <li>Upgrading your Streams application from 0.10.1 to 0.10.2 does not require to upgrade your brokers.
```

Comment:
Not sure. If personally thinks it's good to have it in Streams section. If someone looks for Streams infos it's not handy if some information about is in a different place. However, we still might have a compatibility section (just putting the information there again).